modules/rostests/apitests/kernel32/GetVolumeInformation.c

```diff
@@ -113,8 +113,8 @@ TestGetVolumeInformationW(VOID)
             break;
         }
     }
-    ok(i != MAX_PATH, "String was not null terminated!\n");
-    ok(i >= Len, "Buffer has not been overruned\n");
+    //ok(i != MAX_PATH, "String was not null terminated!\n"); Doesn't work on Windows Server 2003
```

Comment:
If this works on vista+, would be nice to have a version check?
Cc @HeisSpiter as the author of this code

Reply:
Doesn't work on Windows 10 either. Neither Wow64 nor native x64. Seems unlikely to be different on versions in between, but if @HeisSpiter has other information, I'm happy to add the appropriate check. Otherwise I'll invert the check.